dolphinscheduler-common/src/main/resources/common.properties

```
@@ -18,7 +18,7 @@
 # user data local directory path, please make sure the directory exists and have read write permissions
 data.basedir.path=/tmp/dolphinscheduler
 
-# resource storage type: HDFS, S3, NONE
+# resource storage type: HDFS, LOCAL, S3, NONE
 resource.storage.type=NONE
```

I think we should use `LOCAL` as our defalut resource type which our of box and do not need users change config for more user firendly. WDYT @caishunfeng @SbloodyS 
```suggestion
resource.storage.type=LOCAL
```